Adjusted toolbar issue #17705

a23matsh opened the pull request:
Adjusted the toolbar icons sizing, they all now fit 

![image](https://github.com/user-attachments/assets/166874c8-a8e3-4bee-96e4-f586b9c81c68)

a23emijo reviewed (changes requested):
- To start of, my laptops screen is very small compared to other laptops so my operating system recommends me to use 150% zoom in it at all time. This makes the icons appear smaller in the toolbar, I don't think that this is your problem though, but because you moved the icons to the left of the toolbar, a lot of emptiness appears to the right of the icons. A solution to this (which should be another issue) is to make the icons re-size with the toolbar, because the toolbar already re-sizes when you zoom (it gets thinner or thicker) but the icons stay the same size at all time. I also think that the icons should still be in the center like they were before and not to the left. Maybe make the whole toolbar scrollable, but make the scrollbar invisible, instead of moving or re-sizing the icons?

![image](https://github.com/user-attachments/assets/5b2a0159-afce-45f5-b344-12974fec5c9c)

- Why did you remove the white triangle in the bottom right corner of the icons that has a dropdown menu? It was there to indicate that the icon has more icons hidden behind it.

- Why did you remove the "cursor: pointer" in the css? I think it's nicer to have the cursor change to a hand pointing at the object the user hovers. I do like the short transition you added on the icon hover though. 👍

a23matsh commented:
Adjusted the toolbar for the requested changes

-Toolbar still fits all the icons in it without any cut offs
- The cursor is back
- The icons were previously shifted to the left and not the middle since resizing, this is now fixed they remain in the middle

b23danhe commented:
The new issue handling scrollbar is #17756
